lib/core/token_table.h

```diff
@@ -21,68 +21,72 @@
  */
 
 /* DO NOT EDIT! generated by tokens.pl */
-h2o_token_t h2o__tokens[] = {{{H2O_STRLIT(":authority")}, 1, 0, 0, 0, 0},
```

Comment:
doesn't change except for adding `conn-flow-in` and `conn-flow-out`, but re-formatted by `tokens.pl`